test/db/anal/x86_32

```diff
@@ -1005,18 +1005,18 @@ aaa
 CC. @@ switch
 EOF
 EXPECT=<<EOF
-switch table (275 cases) at 0x4140b0
-switch table (121 cases) at 0x413cc0
+switch table (58 cases) at 0x4140b0
```

Comment:
I'm not sure this is what we want. This switch actually has 275 cases, it's just that most of them points to the default case. IDA as well shows "275 cases" and I think it may be misleading seeing less in r2.

Reply:
reverted it